addons/payment_mollie/models/payment_acquirer.py

```diff
@@ -68,7 +68,7 @@ def _mollie_make_request(self, endpoint, data=None, method='POST'):
             "Authorization": f'Bearer {self.mollie_api_key}',
             "Content-Type": "application/json",
             # User agent string used by Mollie
-            "User-Agent": f'Odoo/{odoo_version} MollieOdoo/{module_version}',
+            "User-Agent": f'Odoo/{odoo_version} MollieNativeOdoo/{module_version}',
```

Comment:
```suggestion
            "User-Agent": f'Odoo/{odoo_version} OfficialMollieOdoo/{module_version}',
```

Reply:
@AntoineVDV the string `MollieNativeOdoo` is provided by mollie itself. It is used by mollie's support department. It would be great if we can keep the same string.

Reply:
Ok then, but it's a bit weird that they "provide" that string if it's not used anywhere yet. :raised_eyebrow:

Reply:
I am not sure but most probably their support team differentiates tickets based on this.  BTW, the same string was used in an older version of the module v13, v14 (those apps were external not part of odoo) maybe this could be a reason.

Reply:
Oh ok, so it's the exact same string for the two modules?